creator-node/package.json

```diff
@@ -18,7 +18,8 @@
     "lint:fix": "eslint --format table --fix --ext=js,ts src",
     "lint": "eslint --format table --ext=js,ts src",
     "build": "./node_modules/.bin/tsc --project tsconfig.build.json",
-    "postinstall": "patch-package"
+    "postinstall": "patch-package",
+    "typecheck": "./node_modules/.bin/tsc --project tsconfig.build.json --noEmit"
```

Comment:
how's this diff from build cmd above? just the --noEmit?

Reply:
it's faster and doesn't emit the `build/` dir

Reply:
I got the inspiration for this from the client repo: https://github.com/AudiusProject/audius-client/blob/main/packages/common/package.json

Reply:
i now understand whats happening thx